hack/util.sh

```diff
@@ -570,6 +570,11 @@ function cleanup_openshift {
 	delete_empty_logs
 	truncate_large_logs
 
+	if grep -q 'no Docker socket found' "${LOG_DIR}/openshift.log"; then 
+		# the Docker daemon crashed, we need the logs
+		journalctl --unit docker.service > "${LOG_DIR}/docker.log"
```

Comment:
Can you find the option to limit this to the last four hours of logs?  Otherwise it will mess with people who've been running a long time.

Also, move above the truncate step.